content.go

```diff
@@ -31,12 +32,11 @@ type Content struct {
 	SourceRelationship int               `json:"sourcerelationship,omitempty"` // 0 = indirect, 1 = direct.
 	Length             int               `json:"len,omitempty"`                // Length of content in seconds; appropriate for video or audio.
 	Language           string            `json:"language,omitempty"`           // Content language using ISO-639-1-alpha-2.
+	LanguageB          string            `json:"langb,omitempty"`              // Content language using IETF BCP 47. Only one of language or langb should be present.
 	Embeddable         int               `json:"embeddable,omitempty"`         // Indicator of whether or not the content is embeddable (e.g., an embeddable video player), where 0 = no, 1 = yes.
 	Data               []Data            `json:"data,omitempty"`               // Additional content data.
 	Network            *ChannelEntity    `json:"network,omitempty"`            // Details about the network the content is on.
 	Channel            *ChannelEntity    `json:"channel,omitempty"`            // Details about the channel the content is on.
 	KwArray            []string          `json:"kwarray,omitempty"`            // Array of keywords about the site. Only one of ‘keywords’ or‘kwarray’ may be present.
-	CategoryTaxonomy   CategoryTaxonomy  `json:"cattax,omitempty"`             // Defines the taxonomy in use.
-	LangB              string            `json:"langb,omitempty"`              // Language of the creative using IETF BCP 47. Only one of language or langb should be present.
```

Comment:
would it be possible to revert that? it's a breaking change and would require a major release

Reply:
Ah, nice catch. Sure, I can revert the variable name change.

Reply:
BTW, @dim, since you have not had any release that includes the content obj `langb` introduced in the [linked PR](https://github.com/bsm/openrtb/pull/108), renaming the variable should not break any prior official releases. Let me know if based on that info, its ok to keep the longer `LanguageB` variable name, since it's more consistent with the rest of the codebase.